### ISSUE TITLE
ERR_TOO_MANY_REDIRECTS on login with remember me 

### ISSUE THREAD

**Issue by barilee** [yiisoft/yii2-app-advanced#242]
### What steps will reproduce the problem?

- Create Advanced Application

- Migrate

- Deploy Application on server and migrate database

- Signup and then Login _**with remember me checked**_

### What's expected?

- Log in and redirected to homepage [site/index]

### What do you get instead?

- Login in and redirected to homepage, but the redirection is endless and results in an ERR_TOO_MANY_REDIRECTS error.

### Additional info

If remember me is unchecked. It works just fine.
Also this works perfectly on my local machine. So this problem only occurs on the server.

| Q                | A
| ---------------- | ---
| Yii vesion       | 2.0.11
| PHP version      | 5.6.30
| Operating system | CentOS (Apache)

**Comment by cebe** [comment]
can you give more details about the server configuration? nginx/apache? what is the server config?

**Comment by barilee** [comment]
Thank you for your response.
It's Apache and it's a shared hosting environment.
What sort of configuration would you like me to share? could you direct me please?
I could provide a phpinfo on the server... would that help?

**Comment by cebe** [comment]
not really, the apache config would be more relevant but on shared hosting you might not be able to get that.

**Comment by barilee** [comment]
Wow... that's really painful.
I have disabled autologin and commented out the remember me field and set rememberMe to false in the LoginForm Model.
I'll use it this way for now. 
Thank you so much for your time and help.

**Comment by bbunlock** [comment]
dont suppose you found a fix for this? I am having the exact same problem, tried installing a few versions of yii2, advanced, basic, base, starter kit and no matter what I do I cannot get remember me to work, if I tick it and login its the same endless loop.

my root is public_html but the web folder is one up public_html/web so I have a htaccess file in root pointing everything to web.

When looking at details in chrome webtools I have to delete just the _identity cookie to stop the redirect and I am logged in but via session, session works perfectly every time but the identity cookie is whats stopping my app from working.

Any help would be appreciated.

Kid regards

Wayne

**Comment by fcovasquez** [comment]
I have the same problem and I think that the problem is related with the path or domain of the cookie. I believe that this info could be useful. 

https://github.com/samdark/yii2-cookbook/blob/master/book/cookies.md

Please let me know if you find a solution.

Best regards

**Comment by bbunlock** [comment]
just an update for you all that might be having his same problem, I recently updated my server software via cpanel, I updated the apache and also php, im not sure what version of php I was using before the update but as soon as I went to php 5.6+ I was able to login and no redirect.

it would appear that yii2 apps or at least some require php 5.6 and higher.

once again thanks for the great work on the app.

Kind regards

Wayne
